sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/models/application-offering-change-request.aest.dto.ts

```diff
@@ -0,0 +1,27 @@
+import { ApplicationOfferingChangeRequestStatus } from "@sims/sims-db";
```

Comment:
mm..I think we don't create DTO in cient-type specific files., I believe `application-offering-change-request.institutions.dto.ts` is an uncaught mistake. Correct me if I am wrong team

Reply:
renamed and used the same file for both. Thanks!